login-workflow/example/src/screens/ExampleRegistrationSuccess.tsx

```diff
@@ -9,91 +9,99 @@ type CustomRegistrationProps = {
     registrationData: RegistrationData;
 };
 
-export const CustomRegistrationSuccessScreen: React.FC<CustomRegistrationProps> = (props) => (
-    <>
-        <CardHeader
-            title={
-                <Typography variant={'h6'} style={{ fontWeight: 600 }}>
-                    Custom Registration Success Screen
-                </Typography>
-            }
-            style={{ padding: '16px 24px' }}
-        />
-        <CardContent
-            style={{
-                flex: '1 1 0px',
-                overflow: 'auto',
-                display: 'flex',
-                flexDirection: 'column',
-                justifyContent: 'center',
-                alignItems: 'center',
-                height: '100%',
-                padding: '16px 24px',
-            }}
-        >
-            <EmptyState
-                icon={<Person color={'primary'} style={{ fontSize: 100 }} />}
-                title={`Congratulations ${props.registrationData?.accountDetails?.firstName || ''}${' '}
-                ${props.registrationData?.accountDetails?.lastName || ''}!`}
-                description={'You made it to the custom registration success screen!'}
+export const CustomRegistrationSuccessScreen: React.FC<CustomRegistrationProps> = (props) => {
+    const matchesXS = useMediaQuery('(max-width:600px)');
```

Comment:
Use `theme.breakpoints.down('sm')` here

Reply:
Update: nvm, I just noticed that this is in /example, not the package.

Reply:
Same for the two comments below.

Reply:
Although this might apply to https://github.com/pxblue/react-workflows/pull/47/files?file-filters%5B%5D=.tsx#diff-bc3fffef52a5b621f7f308b0b487583baf19695be838c460641bcb2b0a26414fR48